src/pages/Signup.jsx

```diff
@@ -60,13 +120,18 @@ const Signup = () => {
                 {showPassword ? <EyeOutlined /> : <EyeInvisibleOutlined />}
               </IconButton>
             </div>
-            <Button variant="contained" type="submit" sx={{ backgroundColor: "#0bda73", ":hover": { backgroundColor: "#0ff582" }, boxShadow: "none" }} data-testid="signup-button" onClick={handleSubmit}>
-              Sign up
+            <Button variant="contained" type="submit" sx={{ backgroundColor: "#0bda73", ":hover": { backgroundColor: "#0ff582" }, boxShadow: "none" }} data-testid="signup-button">
```

Comment:
please add a disable property so that when it is loading, the button is disabled.